docker-compose.yaml

```diff
@@ -33,6 +33,30 @@ services:
       labels:
         - "prometheus-job=save-orchestrator"
     logging: *loki-logging-jvm
+  sandbox:
+    image: ghcr.io/saveourtool/save-sandbox:${SANDBOX_TAG}
```

Comment:
Pls also add `save-sandbox:bootBuildImage` to the tasks in `deploy_images.yml` workflow to start publishing this imageF

Reply:
added

Reply:
need to update also https://github.com/saveourtool/save-cloud/blob/master/buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt

Reply:
added